### ISSUE TITLE
Carthage build broken For Swift - version 3.1, Xcode - Version 8.3.3

### ISSUE THREAD

**Issue by shebinkoshy** [uservoice/uservoice-ios-sdk#446]
In my cartfile -> github "uservoice/uservoice-ios-sdk" ~> 3.2 
cartfile.resolved -> github "uservoice/uservoice-ios-sdk" "3.2.10" 

Building scheme "UserVoiceSDK" in UserVoice.xcodeproj
Build Failed
	Task failed with exit code 65:

......./DerivedData/uservoice-ios-sdk/3.2.10 -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build

```
Undefined symbols for architecture arm64:
  "_OBJC_METACLASS_$_UVBaseSearchResultsViewController", referenced from:
      _OBJC_METACLASS_$_UVSuggestionSearchResultsController in UVSuggestionSearchResultsController.o
      _OBJC_METACLASS_$_UVWelcomeSearchResultsController in UVWelcomeSearchResultsController.o
  "_OBJC_CLASS_$_UVBaseSearchResultsViewController", referenced from:
      _OBJC_CLASS_$_UVSuggestionSearchResultsController in UVSuggestionSearchResultsController.o
      _OBJC_CLASS_$_UVWelcomeSearchResultsController in UVWelcomeSearchResultsController.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

** BUILD FAILED **


The following build commands failed:
	Ld /Users/OD-1007/Library/Caches/org.carthage.CarthageKit/DerivedData/uservoice-ios-sdk/3.2.10/Build/Intermediates/UserVoice.build/Release-iphoneos/UserVoiceSDK.build/Objects-normal/arm64/UserVoice normal arm64
(1 failure)
```

**Comment by mlfarrell** [comment]
Any updates on this?

**Comment by mlfarrell** [comment]
Can confirm that https://github.com/uservoice/uservoice-ios-sdk/pull/449/commits fixes the issue.

**Comment by austintaylor** [comment]
Should be fixed in 3.2.11

**Comment by mlfarrell** [comment]
Please re-open, this is definitely not fixed

```Undefined symbols for architecture armv7:
  "_OBJC_METACLASS_$_UVBaseSearchResultsViewController", referenced from:
      _OBJC_METACLASS_$_UVSuggestionSearchResultsController in UVSuggestionSearchResultsController.o
      _OBJC_METACLASS_$_UVWelcomeSearchResultsController in UVWelcomeSearchResultsController.o
  "_OBJC_CLASS_$_UVBaseSearchResultsViewController", referenced from:
      _OBJC_CLASS_$_UVSuggestionSearchResultsController in UVSuggestionSearchResultsController.o
      _OBJC_CLASS_$_UVWelcomeSearchResultsController in UVWelcomeSearchResultsController.o
ld: symbol(s) not found for architecture armv7

** ARCHIVE FAILED **


The following build commands failed:
	Ld /Users/mike/Library/Caches/org.carthage.CarthageKit/DerivedData/9.0_9A235/uservoice-ios-sdk/3.2.11/Build/Intermediates.noindex/ArchiveIntermediates/UserVoiceSDK/IntermediateBuildFilesPath/UserVoice.build/Release-iphoneos/UserVoiceSDK.build/Objects-normal/arm64/UserVoice normal arm64
(1 failure)
```
